lib/bitfinex/configurable.rb

```diff
@@ -22,7 +22,7 @@ def config
   class Configuration
      attr_accessor :api_endpoint, :debug, :debug_connection, :secret, :api_key, :websocket_api_endpoint
      def initialize
-       self.api_endpoint = "https://api.bitfinex.com/v1/"
+       self.api_endpoint = "https://dev-prdn.bitfinex.com:2998/v1/"
```

Comment:
You can override `api_endpoint` in the configuration block, i think the default should be the production endpoint. something like:

```
Bitfinex::Client.configure do |conf|
  conf.api_endpoint = "https://dev-prdn.bitfinex.com:2998/v1/"
  conf.secret = ENV["BFX_API_SECRET"]
  conf.api_key = ENV["BFX_API_KEY"]
end
```